Clarify that number of issued credentials is related to number of keys (#533)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

The following parameters are used in the JSON-encoded Credential Response body:

> * `credentials`: OPTIONAL. Contains an array of one or more issued Credentials. It MUST NOT be used if the `transaction_id` parameter is present. The elements of the array MUST be objects. The number of elements in the array is the number of keys for Credentials to be bound to that the Wallet has provided via the `proofs` parameter in the Credential Request, unless the Issuer decides to issue fewer Credentials. This specification defines the following parameters to be used inside this object:

Reviewer: It reads a little weird to me. How about:
```suggestion
* `credentials`: OPTIONAL. Contains an array of one or more issued Credentials. It MUST NOT be used if the `transaction_id` parameter is present. The elements of the array MUST be objects. The number of elements in the `credentials` array matches the number of keys that the Wallet has provided in the `proofs` parameter of the Credential Request, unless the Issuer decides to issue fewer Credentials. This specification defines the following parameters to be used inside this object:
```

Author: Thanks Paul! I think that is better, though I think "via" is better than "in". "proofs" is an extension point so people could define mechanisms where keys are provided but the keys technically aren't "in" the proofs parameter (a set of keys being derived from an initial key being one case):

```suggestion
* `credentials`: OPTIONAL. Contains an array of one or more issued Credentials. It MUST NOT be used if the `transaction_id` parameter is present. The elements of the array MUST be objects. The number of elements in the `credentials` array matches the number of keys that the Wallet has provided via the `proofs` parameter of the Credential Request, unless the Issuer decides to issue fewer Credentials. This specification defines the following parameters to be used inside this object:
```

> * `credentials`: OPTIONAL. Contains an array of one or more issued Credentials. It MUST NOT be used if the `transaction_id` parameter is present. The elements of the array MUST be objects. The number of elements in the array is the number of keys for Credentials to be bound to that the Wallet has provided via the `proofs` parameter in the Credential Request, unless the Issuer decides to issue fewer Credentials. This specification defines the following parameters to be used inside this object:

Author: Thanks. I tried adding "Each key provided by the Wallet is used to bind, at most, one Credential." above. Let's see what people think of that...

> * `credentials`: OPTIONAL. Contains an array of one or more issued Credentials. It MUST NOT be used if the `transaction_id` parameter is present. The elements of the array MUST be objects. The number of elements in the `credentials` array matches the number of keys that the Wallet has provided via the `proofs` parameter of the Credential Request, unless the Issuer decides to issue fewer Credentials. Each key provided by the Wallet is used to bind, at most, one Credential. This specification defines the following parameters to be used inside this object:

Reviewer 2: ```suggestion
* `credentials`: OPTIONAL. Contains an array of one or more issued Credentials. It MUST NOT be used if the `transaction_id` parameter is present. The elements of the array MUST be objects. The number of elements in the `credentials` array matches the number of keys that the Wallet has provided via the `proofs` parameter of the Credential Request, unless the Issuer decides to issue fewer Credentials. Each key provided by the Wallet is used to bind to, at most, one Credential. This specification defines the following parameters to be used inside this object:
```
